src/aerospike_clj/client.clj

```diff
@@ -160,11 +160,21 @@
     (^void onSuccess [_this ^List records]
       (d/success! op-future records))))
 
-(defn- ^Key create-key [^String aero-namespace ^String set-name ^String k]
-  (when (< MAX_KEY_LENGTH (.length k))
-    (throw (Exception. (format "key is too long: %s..." (subs k 0 40)))))
-  (Key. aero-namespace set-name k))
-
+(defprotocol CreateKey
```

Comment:
That's a funny name for the protocol.
Why not call it `UserKey` or `UserDefinedKey`?